auth-web/src/util/config-helper.ts

```diff
@@ -11,6 +11,10 @@ export default class ConfigHelper {
   static async fetchConfig () {
```

Comment:
may be we should think about adding this helper to common components, so that all app can just re-use.